pkgs/development/python-modules/speedtest-cli/default.nix

```diff
@@ -0,0 +1,31 @@
+{ pkgs
+, buildPythonPackage
+, fetchPypi
+}:
+
+# cannot be built as pythonApplication because the library functions are
+# required for home-assistant
+buildPythonPackage rec {
+  pname = "speedtest-cli";
+  version = "2.0.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "2f3d5aa1086d9b367c03b99db6e3207525af174772d877c6b982289b8d2bdefe";
+  };
+
+  # tests require working internet connection
+  doCheck = false;
+
+  postInstall = ''
+    rm $out/bin/speedtest-cli
+    ln -s $out/bin/speedtest $out/bin/speedtest-cli
```

Comment:
What's the advantage of having a symlink? The additional wrapper doesn't use up much space and I'd prefer following upstream closely as this e.g. facilitates auto-upgrades.

Reply:
In that case, I'd prefer removing `postInstall`.

Reply:
sure, i dropped the commit